### ISSUE TITLE
Add some sort-import rules

### ISSUE THREAD

**Issue by LitoMore** [xojs/eslint-config-xo#120]
How about adding some sort-import rules like these below? Do you have any preferences in this regard?

```js
const xoConfig = [
	{
		rules: {
			'sort-imports': [
				'error',
				{
					ignoreCase: false,
					ignoreDeclarationSort: true,
					ignoreMemberSort: false,
					memberSyntaxSortOrder: ['none', 'all', 'multiple', 'single'],
					allowSeparatedGroups: false,
				},
			],
			'import-x/no-named-as-default': 'off',
			'import-x/extensions': 'off',
			'import-x/order': [
				'error',
				{
					groups: ['builtin', 'external', 'parent', 'sibling', 'index'],
					alphabetize: {
						order: 'asc',
						caseInsensitive: true,
					},
					warnOnUnassignedImports: true,
					'newlines-between': 'never',
				},
			],
		},
	},
];
```

**Comment by sindresorhus** [comment]
Sorting imports by name is just a waste of time. We do already sort for the important things, like type of import.
